### ISSUE TITLE
Training Error

### ISSUE THREAD

**Issue by AmingWu** [MegviiDetection/FSCE#12]
Dear authors,

When I run your code, there is an error. The version of pytorch is 1.4.0 + 0.5.0 torchvision. Could you give me some advice? Thank you.
![QQ截图20210326190809](https://user-images.githubusercontent.com/38445175/112623255-18931480-8e67-11eb-9608-2ff72b719da0.png)

**Comment by Chauncy-Cai** [comment]
It seems like you don't successfully “build".
Plz make sure you have installed all the requirement and successfully executed  "python setup.py build develop".

**Comment by AmingWu** [comment]
Thanks for your reply. I have successfully run your code. However, I do not reproduce your result.

For the 10-shot in Split1, the nAP50 value is 61, which is lower than your report result.

![QQ截图20210328184724](https://user-images.githubusercontent.com/38445175/112749682-815cc700-8ff6-11eb-9a60-a56ea2618752.png)

**Comment by john2020-210** [comment]
@Chauncy-Cai @AmingWu  How do you execute  the  "python setup.py build develop".when I do this ,it report the error "can not find the configs in the model_zoo"

**Comment by bsun0802** [comment]
@Chauncy-Cai Can you please address this?

**Comment by bsun0802** [comment]
> @Chauncy-Cai @AmingWu How do you execute the "python setup.py build develop".when I do this ,it report the error "can not find the configs in the model_zoo"

@Chauncy-Cai Can you please address this? 


> Thanks for your reply. I have successfully run your code. However, I do not reproduce your result.
> 
> For the 10-shot in Split1, the nAP50 value is 61, which is lower than your report result.
> 
> ![QQ截图20210328184724](https://user-images.githubusercontent.com/38445175/112749682-815cc700-8ff6-11eb-9a60-a56ea2618752.png)

@AmingWu I don't know why, I seems to produce nAP > 62.7 easily. 
@Chauncy-Cai  你们还在搞 few-shot 么, 跑一下 VOC 试试, 我觉得我那个 yml 应该是63的点才对啊.我现在没发跑, 离职了没有8卡.

**Comment by Chauncy-Cai** [comment]
> Thanks for your reply. I have successfully run your code. However, I do not reproduce your result.
> 
> For the 10-shot in Split1, the nAP50 value is 61, which is lower than your report result.
> 
> ![QQ截图20210328184724](https://user-images.githubusercontent.com/38445175/112749682-815cc700-8ff6-11eb-9a60-a56ea2618752.png)

I rerun this yaml(10shot split1) last weekend, I do easily reach 62+ in nAP50.

**Comment by Chauncy-Cai** [comment]
> @Chauncy-Cai @AmingWu How do you execute the "python setup.py build develop".when I do this ,it report the error "can not find the configs in the model_zoo"

You can make a soft link to  "FSCE/configs/", this may work.

**Comment by john2020-210** [comment]
@AmingWu  I'm sorry to bother you,I would like to ask about your hardware configuration. I have been reporting errors when running the code.I also use the version of pytorch is 1.4.0 + 0.5.0 torchvision,the graphics card is RTX2080Ti

**Comment by AmingWu** [comment]
@john2020-210 , my hardward configuration is the same to you.  The version of pytorch is 1.4.0 + 0.5.0 torchvision,the graphics card is RTX2080Ti.

**Comment by john2020-210** [comment]
Thanks for you reply ,Does it have anything to do with the system or it has to do with the installation process,my system is ubuntu20.04 .what is yours,and any other matters need attention about installation process?

**Comment by john2020-210** [comment]
@AmingWu  I'm sorry to bother you,.I have successfully run your code. However, I have only 1 GPU on my computer,

For the 10-shot in Split1, the nAP50 value is 57, which is lower than yours ,Could  you share your parameter configuration  ?

**Comment by bsun0802** [comment]
@john2020-210 The results can be reproduced with 8-GPUs with the config we provide only. 

For other then 8-GPUs, we don't know if it can be reproduce or need different configuration.
For example, at least you can refer to learning rate scaling rule (https://stackoverflow.com/questions/53033556/how-should-the-learning-rate-change-as-the-batch-size-change/53046624)

**Comment by Cuzny** [comment]
As "Total number of RoIs per training minibatch = ROI_HEADS.BATCH_SIZE_PER_IMAGE * SOLVER.IMS_PER_BATCH", 
what will happen if I use only 1 GPU but still use the same parameters(BATCH_SIZE_PER_IMAGE and IMS_PER_BATCH) as 8-GPUs'?

**Comment by bsun0802** [comment]
@Cuzny You can refer to https://github.com/MegviiDetection/FSCE/blob/main/fsdet/config/defaults.py for detailed explanation on the behavior of each config parameters.

**Comment by Cuzny** [comment]
@bsun0802 The size of minibatch seems to be 8 times larger?

**Comment by bsun0802** [comment]
@Cuzny I guess, yes, if the GPU memory is enough. But I have no experience training fsdet with 1 GPU and don't know.

**Comment by Cuzny** [comment]
@bsun0802 Maybe I should try to reduce the learning rate to about 1/3. Thanks for your reply.
